thoth/metrics_exporter/jobs.py

```diff
@@ -92,9 +92,12 @@ def count_graph_sync_job_status(job_list_items: list) -> dict:
             graph_sync_jobs_status[job_type]["active"] += 1
         elif "pending" in item["status"].keys():
             graph_sync_jobs_status[job_type]["pending"] += 1
+        elif not item["status"].keys():
```

Comment:
When does this happen?

Reply:
According to metrics ```thoth_graph_sync_jobs_status{job_status="empty",job_type="graph-sync-solver",namespace="thoth-middletier-stage"} 7.0```

Reply:
Not many times. But in the log you can spot it sometimes: ```

2019-06-21 12:48:11,852 [1] ERROR    thoth.metrics_exporter.jobs:96: Unknown job status {}
--
  | 2019-06-21 12:48:11,881 [1] ERROR    thoth.metrics_exporter.jobs:96: Unknown job status {'conditions': [{'lastProbeTime': '2019-06-21T10:55:21Z',
  | 'lastTransitionTime': '2019-06-21T10:55:21Z',
  | 'message': 'Job has reached the specified backoff limit',
  | 'reason': 'BackoffLimitExceeded',
  | 'status': 'True',
  | 'type': 'Failed'}],
  | 'startTime': '2019-06-21T10:40:22Z'}

```

Reply:
It might be a good idea to reuse logic we have in openshift.py for this and ask openshift class for status report so we handle these cases only once - https://github.com/thoth-station/common/blob/adb7515b8383524bd9103c11cc3d8a149e4416b6/thoth/common/openshift.py#L433